INSTALL.md

```diff
@@ -73,13 +73,12 @@ sudo zypper install dkms-dattobd dattobd-utils
 sudo apt-key adv --recv-keys --keyserver keys.fedoraproject.org 370C85D709D26407
 echo "deb [arch=amd64] https://cpkg.datto.com/datto-deb/public/$(lsb_release -sc) $(lsb_release -sc) main" | sudo tee /etc/apt/sources.list.d/datto-linux-agent.list
 sudo apt-get update
-sudo apt-get install dattobd-dkms dattobd-utils
+sudo apt-get install linux-headers-$(uname -r) dattobd-dkms dattobd-utils
```

Comment:
This should be installed as a dependency of DKMS. Is there a reason for explicitly stating here?